### DISCUSSION TITLE
Elemental Merge helper control

### DISCUSSION THREAD

**Discussion by wkpark** [hako-mikan/sd-webui-supermerger#225] [General]
This is a new Elemental Merge helper control (support SD + SDXL)

I'm not sure it is helpful, but It could be a bit easier than without this helper.

(blocks  -> elements map will be auto-generated and cached.)

See also https://github.com/hako-mikan/sd-webui-supermerger/blob/main/elemental_en.md#elemental

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/18f1fc40-4687-49b0-ad3a-e1441abc8ff9)

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/921a62cc-77af-44a9-9af3-bb0bf10bf934)

**Comment by groinge** [comment]
Haha i was just working on this exact feature when i stumbled on to this thread

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/124379871/65e606f4-3b3e-43ed-bd04-04043e4106e2)

Did Hako-mikan deny your PR?

**Reply by wkpark** [reply]
![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/659fad9a-4f69-42d3-a41c-2dcdadb35532)

![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/232347/5a2509bb-ffa4-4a94-b6c3-9678410e321e)


I didn't submit a PR because the control seemed too complicated. and above screenshot is the updated control (include "NOT" checkbox)

**Reply by groinge** [reply]
Looks good. I'm not sure if its possible to make elemental merging not complicated.
